models/ac_generator.h

```diff
@@ -128,12 +164,20 @@ class ac_generator : public Node
     double A_11_;
   };
 
+  double
+  get_I_() const
+  {
+    return S_.I_;
```

Comment:
@gtrensch No. If you remove `S_.I_` entirely, you would also delete line 225 above.